open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 ## Walking Down the Path: Branching Logic
 
 <img alt="Woman at crossroads" src={useBaseUrl('img/einar-storsul-K3W7I7x37Xk-unsplash.jpg')} />
+
 _Photo by Einar Storsul on Unsplash_
 
-All but the most complex Docassemble interviews will have some choices that change the information
-that you show to the user or that you will need to ask a follow-up question for. Sometimes, this choice
-making is called branching logic. It's the kind of choice that you might think of making with a flowchart.
+Very simple interviews might just ask a user some questions and spit out some result. Most interviews, though, will want to change what the questions they ask and the results they show based on a user's answers.
+
+Our code has to make a choice and, until the AI take over, we have to give it very specific instructions about how to make that choice. That's often called *branching logic*. Probably because of trees.
 
-In today's class, we'll talk about how to write a Docassemble interview that achieves this goal.
+## Example 1
```

Comment:
I think this may end up being the only example. If so, the name of this heading should change.